### ISSUE TITLE
This still a thing?

### ISSUE THREAD

**Issue by noobtoob4lyfe** [iBobbyTS/OpenVideoEnhance#1]
This looks pretty interesting but I can't figure out how to use it.  Is the documentation still in the works?  thanks!

**Comment by iBobbyTS** [comment]
I've been working on my school stuff, I'll work out the documentation within a week

**Comment by shukkkur** [comment]
@iBobbyTS yes, please ) I look forward to detailed, user-friendly, step-by-step guide to be able to run on videos and frames
